kgforge/specializations/models/rdf/service.py

```diff
@@ -187,23 +209,18 @@ def generate_context(self) -> Dict:
         raise NotImplementedError()
 
     @abstractmethod
-    def _build_shapes_map(self) -> Dict:
-        """Queries the source and returns a map of owl:Class to sh:NodeShape"""
+    def _build_shapes_map(self) -> Tuple[Dict, Dict, Dict]:
```

Comment:
Looks like a Dict is missing. Both implementations return 4 dictionaries. The return type should change + the doc

Reply:
done